src/Connection.cpp

```diff
@@ -45,9 +46,14 @@ Connection::~Connection() {
 int Connection::init(const char* host, uint32_t port, const char* alias) {
   snprintf(m_host, sizeof m_host, "%s", host);
   m_port = port;
+  m_local = m_host[0] == '/'; // un.h UNIX_PATH_MAX < netdb.h NI_MAXHOST
```

Comment:
This method of socket type detection fails for windows paths

Reply:
The mistake highlights a downside of host string polymorphism, but it makes the language bindings easier

Reply:
Given the path detection also has to be used in the python and go bindings, I'm going to export a C++ function to do it

Reply:
not to mention relative paths

Reply:
Oh, I forgot my initial shorthand came from [pylibmc's rule](https://sendapatch.se/projects/pylibmc/reference.html)

Reply:
Alright, I'm closing this thread seeing as the platform incompatibility seems standardized. Everything should probably be in docker/WSL anyways. Overall PR is still a work in progress but wanted a commit with the python server string parsing version in case I end up wanting it for unit tests. I'm assuming it's squash and merge so my use of the PR as a notepad won't add clutter.